OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/operations/impl/executors/LoginUserOperationExecutor.kt

```diff
@@ -160,7 +164,18 @@ internal class LoginUserOperationExecutor(
 
         try {
             val subscriptionList = subscriptions.toList()
-            val response = _userBackend.createUser(createUserOperation.appId, identities, subscriptionList.map { it.second }, properties)
+            val pushSubscription = subscriptions.values.find { it.type == SubscriptionObjectType.ANDROID_PUSH }
```

Comment:
This SDK supports fireOS and Huawei push so you will need a more complete check. However, since this looks like it is for Device Auth then we can remove it.
* That is, once we have 100% committed to that path, so hold on addressing this comment until then.

Reply:
The only purpose for the pushSubscription is to acquire the push token. Since we are confirmed that we don't need the token for subscription operation, I removed it in the fixup commit.

Reply:
I see you removed the usage of this but this line was not cleaned up.